pkg/controller/migmigration/description.go

```diff
@@ -0,0 +1,53 @@
+	"StagePodsCreated":                "Waiting for all Stage Pods to start.",
+	"StagePodsFailed":                 "Migration failed due to some Stage Pods failing to start.",
+	"RestartRestic":                   "Restarting Restic Pods, ensuring latest PVC mounts are available for PVC backups.",
+	"ResticRestarted":                 "Waiting for Restic Pods to restart, ensuring latest PVC mounts are available for PVC backups.",
+	"QuiesceApplications":             "Quiescing (Scaling to 0 replicas): Deployments, DeploymentConfigs, StatefulSets, ReplicaSets, DaemonSets, CronJobs and Jobs.",
+	"EnsureQuiesced":                  "Waiting for Quiesce (Scaling to 0 replicas) to finish for Deployments, DeploymentConfigs, StatefulSets, ReplicaSets, DaemonSets, CronJobs and Jobs.",
+	"UnQuiesceApplications":           "UnQuiescing (Scaling to N replicas) Deployments, DeploymentConfigs, StatefulSets, ReplicaSets, DaemonSets, CronJobs and Jobs.",
+	"EnsureStageBackup":               "Creating 'Stage' Backup.",
+	"StageBackupCreated":              "Waiting for completion of 'Stage' Backup.",
+	"StageBackupFailed":               "Migration failed during 'Stage' Backup.",
+	"EnsureInitialBackupReplicated":   "Waiting for 'initial' Velero Backup replication to target cluster.",
+	"EnsureStageBackupReplicated":     "Waiting for 'initial' Velero Backup replication to target cluster.",
+	"EnsureStageRestore":              "Creating 'Stage' Velero Restore including OpenShift resources and PVCs.",
+	"StageRestoreCreated":             "Waiting for completion of 'Stage' Velero Restore.",
+	"StageRestoreFailed":              "Migration failed during 'Stage' Velero Restore.",
+	"EnsureFinalRestore":              "Creating 'Final' Velero Restore.",
+	"FinalRestoreCreated":             "Waiting for completion of 'Final' Velero Restore.",
+	"FinalRestoreFailed":              "Migration failed during 'Final' Velero Restore.",
+	"Verification":                    "Verifying health of migrated Pods.",
+	"EnsureStagePodsDeleted":          "Deleting any leftover Stage Pods.",
+	"EnsureStagePodsTerminated":       "Waiting for leftover Stage Pod deletion to finish.",
+	"EnsureAnnotationsDeleted":        "Removing migration annotations and labels from PVs, PVCs, Pods, ImageStreams, and Namespaces. Annotations and labels provide migration instructions to Velero, Velero Plugins and Restic.",
+	"EnsureMigratedDeleted":           "Rolling back. Waiting for migrated resource deletion.",
+	"DeleteMigrated":                  "Rolling back. Deleting migrated resources from target cluster.",
+	"DeleteBackups":                   "Deleting Velero Backups created during migration.",
+	"DeleteRestores":                  "Deleting Velero Restores created during migration.",
+	"MigrationFailed":                 "Migration failed.",
+	"Canceling":                       "Migration cancellation in progress.",
+	"Canceled":                        "Migration cancelled.",
```

Comment:
Looks like I need to decide how I want to spell "cancelled / canceled" :)

Reply:
@djwhatle "canceled" is the preferred spelling in American English and the one I've been directed to use in UIs.

Reply:
@djwhatle Thanks for doing this - it will be a nice UX enhancement. How do you want me to comment on the text - in the PR, or can I send you a file with recommended changes?

Reply:
@vconzola whatever works best for you!